modules/ollama/options.go

```diff
@@ -37,3 +47,34 @@ func withGpu() testcontainers.CustomizeRequestOption {
 		}
 	})
 }
+
+var _ testcontainers.ContainerCustomizer = (*UseLocal)(nil)
+
+// UseLocal will use the local Ollama instance instead of pulling the Docker image.
+type UseLocal struct {
+	env []string
+}
+
+// WithUseLocal the module will use the local Ollama instance instead of pulling the Docker image.
+// Pass the environment variables you need to set for the Ollama binary to be used,
+// in the format of "KEY=VALUE". KeyValue pairs with the wrong format will cause an error.
+func WithUseLocal(keyVal ...string) UseLocal {
```

Comment:
question: would it be better to have this passed as a map otherwise people have to manually construct these and we have to parse for Customize.

Reply:
I initially thought about that, and chose this approach not to force passing a nil map. But removing the parsing seems like a good compromise

Reply:
Ahh I like not having to pass nil as you say, if this was key value pairs you get the best of both, just having to do a little work here to store